providers/aws/systemsmanager/managedinstances.go

```diff
@@ -0,0 +1,91 @@
+package systemsmanager
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"github.com/tailwarden/komiser/models"
+	"github.com/tailwarden/komiser/providers"
+)
+
+func getManagedEc2(ctx context.Context, client providers.ProviderClient) ([]models.Resource, error) {
+	resources := make([]models.Resource, 0)
+
+	ssmClient := ssm.NewFromConfig(*client.AWSClient)
+	ec2Client := ec2.NewFromConfig(*client.AWSClient)
+
+	ssmOutput, err := ssmClient.DescribeInstanceInformation(ctx, &ssm.DescribeInstanceInformationInput{
+		MaxResults: aws.Int32(50),
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	instanceIds := make([]string, 0, len(ssmOutput.InstanceInformationList))
+	for _, ec2instance := range ssmOutput.InstanceInformationList {
+		instanceIds = append(instanceIds, *ec2instance.InstanceId)
+	}
+	ec2Output, err := ec2Client.DescribeInstances(ctx, &ec2.DescribeInstancesInput{
+		InstanceIds: instanceIds,
+	})
+	if err != nil {
+		return nil, err
```

Comment:
```suggestion
		return resources, err
```

some nits for readability